src/scripts/bee.js

```diff
@@ -241,20 +250,34 @@ class Bee extends MovingObject {
     );
     let velocityPreLaunch = this.launched ? this.vel : Bee.START_VEL;
 
+    let beeDirection = this.beeDirection();
+    let mapping = {
+      right: this.beeRightFrames,
+      left: this.beeLeftFrames,
+      up: this.beeUpFrames,
+      down: this.beeDownFrames,
+    };
+    animationSequence = mapping[beeDirection];
+
+    return animationSequence[beeFrameIdx];
+  }
+
+  beeDirection() {
+    let velocityPreLaunch = this.launched ? this.vel : Bee.START_VEL;
+
     if (Math.abs(velocityPreLaunch[0]) >= Math.abs(velocityPreLaunch[1])) {
       if (velocityPreLaunch[0] >= 0) {
-        animationSequence = this.beeRightFrames;
+        return "right";
       } else {
-        animationSequence = this.beeLeftFrames;
+        return "left";
       }
     } else {
       if (velocityPreLaunch[1] <= 0) {
-        animationSequence = this.beeUpFrames;
+        return "up";
       } else {
-        animationSequence = this.beeDownFrames;
+        return "down";
```

Comment:
This is what we call "stringly typed" - using strings as loose types can be prone to errors/mistakes.

One way to avoid this is to create an object w/ properties that you can reference.

E.g.
```
const Animals = {
  DOG = 'dog',
  CAT = 'cat',
  MOUSE = 'mouse',
}

getAnimal() {
  return Animal.DOG;
}

if (getAnimal() === Animal.CAT) {
  return 'YAYAYAYAYA';
}
return 'OYOYOYOY';
```

Reply:
Good point, danger. Refactoring to use objects w properties instead.